ui/cap-react/src/components/drafts/form/themes/grommet/components/ImportLayer.js

```diff
@@ -44,45 +190,55 @@ class ImportLayer extends React.Component {
             wrap={false}
             flex={true}
           >
-            <Box flex={true}>
-              <Box pad={{ vertical: "small" }}>{this.pasteDesctiption}</Box>
+            {this.state.showTextArea && (
```

Comment:
If `this.props.options.listSuggestions` is missing, we should render automatically the textarea